url/interfaces.any.js

```diff
@@ -28,7 +21,7 @@
   USVString toJSON();
 };
 
-[Constructor(optional (USVString or URLSearchParams) init = ""),
+[Constructor(optional (sequence<sequence<USVString>> or USVString) init = ""), // or record<USVString, USVString>
```

Comment:
I do, but I'm still here for a day or two, and if you ask now I can take a look.